sklearn/tests/test_pipeline.py

```diff
@@ -1264,3 +1264,11 @@ def test_feature_union_warns_unknown_transformer_weight():
     union = FeatureUnion(transformer_list, transformer_weights=weights)
     with pytest.raises(ValueError, match=expected_msg):
         union.fit(X, y)
+
+
+@pytest.mark.parametrize('passthrough', [None, 'passthrough'])
+def test_pipeline_get_tags_none(passthrough):
```

Comment:
Is this a non-regression test for https://github.com/scikit-learn/scikit-learn/issues/18815 ? If so let's write  it ;)